### PR TITLE
release notes for 2024.10.10

### PR DESCRIPTION
**Pull request by BurnzZ**

### CODE REVIEW

**Review by wRAR — APPROVED** [review]

**Review by PyExplorer — APPROVED** [review]

**Comment by kmike** [comment]
Do you think we can switch to date-based versioning in this release?

**Comment by BurnzZ** [comment]
> Do you think we can switch to date-based versioning in this release?

Thanks for reminding me on this @kmike . What do you think about using the `YYYY.MM.DD` format? Where the date represents the release date of the package.

**Comment by kmike** [comment]
@BurnzZ yes, seems fine, it's one of the common https://calver.org/ formats.

We just need to ensure that version sorting works properly, i.e. the new version should be larger than the existing 0.3, according to pip. But it seems there won't be an issue here.

**Comment by kmike** [comment]
Also, we'd need to decide how to handle multiple releases in one day, but we can always add something to the end of the version.

**Comment by BurnzZ** [comment]
> the new version should be larger than the existing 0.3, according to pip. But it seems there won't be an issue here.

Yeah, it seems everything seems to be given that `YYYY.*` has a higher value than `Major.*`

> Also, we'd need to decide how to handle multiple releases in one day, but we can always add something to the end of the version.

I suppose something like `YYYY.MM.DD_MICRO` or ` YYYY.MM.DD.MICRO` can be used later on.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+2024.10.10 (2024-10-10)
+-----------------------
+
+* Rules for ``queryRemoval`` got 5 new domains and 6 existing domains have
+  new parameters that were added.
```
Comment (kmike) [inline CHANGES.rst:8]: We can also mention that the versioning is changed to calver

**Review by kmike — APPROVED** [review]

**Pull request merged**